requests/maniphest_search.go

```diff
@@ -0,0 +1,116 @@
+package requests
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/uber/gonduit/entities"
+)
+
+// ManiphestSearchRequest represents a request to maniphest.search API method.
+type ManiphestSearchRequest struct {
+	// QueryKey is builtin or saved query to use. It is optional and sets initial constraints.
+	QueryKey    string             `json:"queryKey,omitempty"`
+	// Constraints contains additional filters for results. Applied on top of query if provided.
+	Constraints *SearchConstraints `json:"constraints,omitempty"`
+	// Attachments specified what additional data should be returned with each result.
+	Attachments *SearchAttachments `json:"attachments,omitempty"`
+
+	*entities.Cursor
+	Request
+}
+
+// SearchAttachments contains fields that specify what additional data should be returned with sesarch results.
+type SearchAttachments struct {
```

Comment:
Please rename to `ManiphestSearchAttachements` as there are other `*.search` methods which might be implemented sooner or later.